package.json

```diff
@@ -17,21 +17,22 @@
     "sax": ">=0.6",
     "selectn": "^0.9.6",
     "strip-bom": "~0.3.1",
-    "xml-crypto": "~0.8.0"
+    "xml-crypto": "~0.8.0",
+    "ursa": "0.8.5 || >=0.9.4"
   },
   "optionalDependencies": {
     "ursa": "0.8.5 || >=0.9.4"
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/milewise/node-soap.git"
+    "url": "git+https://github.com/omerson/node-soap"
   },
   "main": "./index.js",
   "directories": {
-    "lib": "./lib"
+    "test": "test"
   },
   "scripts": {
-    "toc": "./node_modules/.bin/doctoc Readme.md --github --maxlevel 3",
+    "toc": "doctoc Readme.md --github --maxlevel 3",
```

Comment:
why did you change the script command? this would implicit that the user has `doctoc` installed globally - otherwise it would work with the `doctoc` lib installed in `soap`s `node_modules/` directory